lecture-notes/3-bld.md

```diff
@@ -680,31 +680,271 @@ $
 
 you can use `HELLO` as a Make variable.
 
-In short, environment variables (e.g. `export CFLAGS=-g ; make hello`):
+In short, when using environment variables in commands such as `export CFLAGS=-g ; make hello`:
 
-* Tells the shell to run make with `CFLAGS` set in the environment
+* The shell runs make with `CFLAGS` set in the environment
 * Make imports all environment variables as Make variables
 * However, assignments in the Makefile override environment variables! `=` and
   `:=` replace them entirely, while `+=` appends to them
 
+So, for example, the following Makefile overrides the environment variable:
+
+```make
+.PHONY: all
+HELLO:=Max
```

Comment:
Do we want a convention with spaces around the assignment? That's what I typically see, but I'm fine either way as long as we're consistent.

Reply:
TODO mention whitespace stripping on left and right